phoenicis-library/src/main/java/org/phoenicis/library/dto/ShortcutCategoryDTO.java

```diff
@@ -79,19 +88,27 @@ public boolean equals(Object o) {
 
         ShortcutCategoryDTO that = (ShortcutCategoryDTO) o;
 
-        return new EqualsBuilder().append(name, that.name).append(description, that.description)
+        return new EqualsBuilder().append(id, that.id).append(name, that.name).append(description, that.description)
                 .append(shortcuts, that.shortcuts).append(icon, that.icon).isEquals();
     }
 
     @Override
     public int hashCode() {
-        return new HashCodeBuilder(17, 37).append(name).append(description).append(shortcuts).append(icon).toHashCode();
+        return new HashCodeBuilder(17, 37).append(id).append(name).append(description).append(shortcuts).append(icon)
+                .toHashCode();
+    }
+
+    @Override
+    public void translate() {
+        this.name = tr(this.name);
+        this.description = tr(this.description);
     }
 
     @JsonPOJOBuilder(buildMethodName = "build", withPrefix = "with")
     public static class Builder {
-        private String name;
-        private String description;
+        private String id = "";
+        private String name = "";
+        private String description = "";
```

Comment:
Yes, this is required anyway because one could call the builder with null arguments.
You can also take this into account directly in tr(String) method